horace_core/sqw/file_io/@sqw_binfile_common/private/extract_correct_subobj_.m

```diff
@@ -23,7 +23,7 @@
     if strcmp(obj_name, 'header')
         subobj = input_obj.experiment_info;
     elseif strcmp(obj_name, 'detpar')
-        subobj = input_obj.detpar();
+        subobj = input_obj.detpar;
```

Comment:
again, `detpar` is rewired to  detector array. Why do you need this change?

Reply:
detpar is a dependent variable. The function call () is and alway was an error. If anything still wishes to extract a detpar subobj, which is now th detector_arrays, this will do it